### ISSUE TITLE
[HOLD for payment 2025-02-04] [HOLD for payment 2025-01-28] [$250] Web - Trip - Payment with Expensify button shown next to submit button on Open trip report

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#54538]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.78-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://github.com/Expensify/App/pull/52823
**Email or phone of affected tester (no customers):** applausetester+0948501@applause.expensifail.com
**Issue reported by:** Applause Internal Team
**Device used:** Windows 11 / Chrome
**App Component:** Chat Report View

## Action Performed:
Precondition:
- Create a trip with steps from this PR https://github.com/Expensify/App/pull/52823

1. Go to staging.new.expensify.com
2. Go to trip room.
3. Click on the expense preview.

## Expected Result:
There should only be a Submit button at the top of the report page.

## Actual Result:
There are two buttons - "Pay with Expensify" and "Submit"

## Workaround:
Unknown

## Platforms:
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/44d8c17a-72bd-4d65-b959-59eefa4b3fe8

![Bug6701759_1735043098161!bandicam_2024-12-24_20-22-09-016](https://github.com/user-attachments/assets/a3c54aff-0303-4e06-b06d-c2c8f3e4826c)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021876334627155535758</li>
        <li>Upwork Job ID: 1876334627155535758</li>
        <li>Last Price Increase: 2025-01-06</li>
<li>Automatic offers: </li>
<ul>
<li>shubham1206agra | Contributor | 105583255</li>
</ul></li>
    </ul>
</details>

**Comment by sakluger** [comment]
The actual bug here is that the expense report shows a `Pay with expensify` button at all. The report is still open, so it shouldn't have a Pay option, in fact, I can't think of any scenario where we would show both a submit and pay button at the same time.

@rojiphil @jacobkim9881 do you think this is related to https://github.com/Expensify/App/issues/53226 at all?

**Comment by jacobkim9881** [comment]
@sakluger I not sure if it is related. I have tried `Action Performed` but I couldn't because I can't enable travel. I can test if I get a company email.
![스크린샷, 2024-12-27 10-59-05](https://github.com/user-attachments/assets/fafc2fe6-7fee-445b-a94a-cbde85c6ee90)

**Comment by sakluger** [comment]
@jacobkim9881 we can't get you a company email, but I can add your testing email to the travel beta. What email are you using to test?

**Comment by sakluger** [comment]
Actually, it looks like you would have to actually book travel to test, which would require you to spend real money. I'm going to ask around internally first.

**Comment by jacobkim9881** [comment]
> Actually, it looks like you would have to actually book travel to test, which would require you to spend real money. I'm going to ask around internally first.

I understood. I will wait for internal team.

**Comment by jacobkim9881** [comment]
> we can't get you a company email, but I can add your testing email to the travel beta. What email are you using to test?

I know I should actually book travel to test but in case my email is like below

**Comment by jacobkim9881** [comment]
jacobkim9881+173@gmail.com

**Comment by sakluger** [comment]
I asked for help in Slack here to see if this is external or internal: https://expensify.slack.com/archives/C05S5EV2JTX/p1735839407611239

**Comment by sakluger** [comment]
Hey @jacobkim9881 - I set jacobkim9881+173@gmail.com as a Travel test account. Please go to your [account settings in Expensify Classic](https://staging.expensify.com/settings?param={%22section%22:%22account%22}) and add a first and last name.

Then, to book fake travel:

1. Go to Travel in the LHN
2. Click the "Book or manage Travel" button
3. Check the checkbox to accept terms
4. Enter an address (it can be fake)
5. Click confirm and wait
6. You will then be logged into Spotnana

Then, Inside of Spotnana:

1. Search for a new booking
   1. For flights:
      - This can be flaky, so it is recommended to use the following flight
      - One way from SEA to SFO
      - Alaska Airlines
      - At least 1 month in the future
   2. For hotels:
      1. Search for any major airport (usually SFO, NYC, LHR)
      2. Book any of top chains (usually Marriott, Hilton)
2. Select the flight
3. Add the other required information and click save at the top of the panel
4. Click "next" on the right side
5. Click "select a trip" and create a new trip
6. Add the [stripe test card](https://docs.stripe.com/testing) ending in 4444. All data such as address, name etc. can be fake, as long as the expiration is after the current date
7. Book the trip

**Comment by sakluger** [comment]
Once you book the fake travel, you should see the trip report in NewDot.

**Comment by jacobkim9881** [comment]
@sakluger I understood and let me test [these step](https://github.com/Expensify/App/issues/54538#issuecomment-2568542837).

**Comment by jacobkim9881** [comment]
@sakluger Thanks for guiding me with the steps. I have added a first and last name at account setting in OD. I couldn't find "Book or manage Travel" button at Travel in LHN of OD but I have this screen. 
![스크린샷, 2025-01-03 11-12-58](https://github.com/user-attachments/assets/3e13002a-cbc2-4f1b-a65a-bf0b7d23416d)

**Comment by sakluger** [comment]
Sorry @jacobkim9881 - I forgot to add you to the Travel beta. I've added you now, and so the button should show up within the next hour (betas can take up to 1 hour to be applied). Let me know if you run into any other issues.

**Comment by sakluger** [comment]
@jacobkim9881 thanks for trying! Sorry you weren't able to get it working without a company email.

@shubham1206agra and @blazejkustra I will assign you two as suggested.

**Comment by sakluger** [comment]
@blazejkustra could you please comment on the issue so that I can assign you?

**Comment by blazejkustra** [comment]
Sorry, I was OOO until today. Commenting for assigment @sakluger

**Comment by blazejkustra** [comment]
@IuliiaHerets I can't reproduce it, all my trip previews don't have 'Submit' button, maybe your workspace is configured in another way 🤔 

<img width="645" alt="image" src="https://github.com/user-attachments/assets/3bc0a17b-9e39-4e98-bea0-98ba33931b42" />

Could you add me to that workspace and make me an admin? Here is my email: `blazej.kustra+trav3@swmansion.com`

**Comment by stitesExpensify** [comment]
Nope, that makes sense to me! I just wanted to make sure we were pointing @blazejkustra in the right direction

**Comment by sakluger** [comment]
Got it, sounds good!

**Comment by sakluger** [comment]
Hey @shubham1206agra - please complete the BZ checklist so that I can complete payment. Thanks!

**Comment by sakluger** [comment]
Thanks!

Summarizing payment on this issue:

Contributor: @blazejkustra - No payment needed
Contributor+: @shubham1206agra $250, paid via Upwork
